### DISCUSSION TITLE
502 error with SvelteKit SSR and caprover

### DISCUSSION THREAD

**Discussion by autoklasa** [caprover/caprover#1597] [Q&A] [answered]
I have weird error with SvelteKit setup on caprover, when I enable SSR and only in production. One of pages (which loads big components server side) throws 502 only on hard refresh.

Thanks to awesome caprover community I found how to trace logs of my applications and nginx containers and I was able to pin point that above mentioned reqests results in this entry in nginx log `upstream sent too big header while reading response header from upstream` can you advice on how to fix this and which config file would be the best to make modifications?

This is only config i found that is supposed to "fix" this kind of issue:
```
~/.config/valet/Nginx/all.conf
proxy_buffer_size   128k;
proxy_buffers   4 256k;
proxy_busy_buffers_size   256k;
fastcgi_buffers 16 16k;
fastcgi_buffer_size 32k;
fastcgi_busy_buffers_size 32k;
```

any advice would be highly apreciated <3

**Comment by autoklasa** [comment, marked as answer]
I have calculated header size to be 4k - it is response link headers.
and have added this: proxy_buffer_size  16k; to app nginx config

caprover+sveltekit KING
<3

**Comment by Antoine-lb** [comment]
More info on this issue: https://github.com/sveltejs/kit/issues/11084
